tests/build_/test_plugins_configuration.py

```diff
@@ -887,6 +889,22 @@ def test_render_resolve_remote_sources(self, build_type, dependency_replacements
             with pytest.raises(NoSuchPluginException):
                 assert get_plugin(plugins, 'prebuild_plugins', 'resolve_remote_source')
 
+    @pytest.mark.parametrize('build_type', (BUILD_TYPE_ORCHESTRATOR, BUILD_TYPE_WORKER))
+    def test_render_add_image_content_manifest(self, build_type):
+        phase = 'prebuild_plugins'
+        plugin = 'add_image_content_manifest'
+        user_params = get_sample_user_params(build_type=build_type)
+        build_json = PluginsConfiguration(user_params).render()
+        plugins = get_plugins_from_build_json(build_json)
+        if build_type == BUILD_TYPE_ORCHESTRATOR:
+            with pytest.raises(NoSuchPluginException):
+                assert get_plugin(plugins, phase, plugin)
+        else:
+            assert get_plugin(plugins, phase, plugin)
+            plugin_args = plugin_value_get(plugins, phase, plugin, 'args')
+            if plugin_args.get('remote_source_icm_url'):
+                assert plugin_args.get('remote_source_icm_url') == TEST_REMOTE_SOURCE_ICM_URL
```

Comment:
Is this being executed? Should the `remote_source_icm_url` be parametrized here as with other args, such as in https://github.com/containerbuildsystem/osbs-client/blob/62f7c90284f10b51dc669dc8c6deb2ab8422736f/tests/build_/test_plugins_configuration.py#L844 ?